scripts/fhir-works-deploy.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -e
```

Comment:
I tried using `yarn` to add local packages, but `yarn` kept caching the local packages. Therefore, I was not always getting the latest packages when trying to deploy. Eventually I gave up using yarn local packages, and used yalc instead. [Reference](https://github.com/yarnpkg/yarn/issues/2165)